google/cloud/internal/timer_queue_test.cc

```diff
@@ -88,7 +88,8 @@ TEST(TimerQueueTest, ScheduleMultipleRunners) {
   std::vector<future<std::thread::id>> futures(kTimers);
   auto const now = std::chrono::system_clock::now();
   std::generate(futures.begin(), futures.end(), [&, now] {
-    return tq.Schedule(now + std::chrono::seconds(1)).then([](auto) {
+    return tq.Schedule(now).then([](auto) {
+      std::this_thread::sleep_for(std::chrono::milliseconds(1));
```

Comment:
While I'm sure this helps, it stills seems like there is no guarantee that a single thread doesn't yield all values.

Reply:
Agree.  But the probability seems to be less than 1 in 10,000 😐